src/extensions/clusters/IndividualClustersView/IndividualClusterView.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FunctionComponent, useCallback, useMemo } from 'react';
+import { createCalculationPool, HitherJobStatusView, useHitherJob } from '../../common/hither';
+import { Recording, Sorting, SortingSelection, SortingSelectionDispatch } from '../../extensionInterface';
+import IndividualClusterWidget from './IndividualClusterWidget';
+
+type Props = {
+    recording: Recording
+    sorting: Sorting
+    selection: SortingSelection
+    selectionDispatch: SortingSelectionDispatch
+    unitId: number
+    width: number
+    height: number
+}
+
+const calculationPool = createCalculationPool({maxSimultaneous: 6})
+
+type Result = {
+    timepoints: number[]
+    x: number[]
+    y: number[]
+}
+
+const IndividualClusterView: FunctionComponent<Props> = ({ recording, sorting, selection, selectionDispatch, unitId, width, height }) => {
+    const {result: features, job} = useHitherJob<Result>(
+        'createjob_individual_cluster_features',
+        {
+            recording_object: recording.recordingObject,
+            sorting_object: sorting.sortingObject,
+            unit_id: unitId
+        },
+        {
+            useClientCache: true,
+            calculationPool
+        }
+    )
+    const selectedIndex: number | undefined = useMemo(() => {
+        const t = selection.currentTimepoint
+        if (t === undefined) return undefined
+        if (features === undefined) return undefined
```

Comment:
@jsoules It's tricky syntax. It's not a type annotation, the output is call result, but I renamed it to features. The type is Result because that's the generic of the useHitherJob hook:

```typescript
const {result: features, job} = useHitherJob<Result>(
        'createjob_individual_cluster_features',
        {
            recording_object: recording.recordingObject,
            sorting_object: sorting.sortingObject,
            unit_id: unitId
        },
        {
            useClientCache: true,
            calculationPool
        }
    )
```